### ISSUE TITLE
patterns request=random throws keyword error

### ISSUE THREAD

**Issue by robintwhite** [juangallostra/Colourlovers-API-wrapper#29]
cl = clapi.ColourLovers()
pattern = cl.search_patterns(request='random')[0]

## Current behavior
KeyError: 'request'

## Expected behavior
'new' and 'top' return correct result

**Comment by juangallostra** [comment]
@robintwhite I tried to but haven't been able to reproduce the issue. The traceback you are getting is for `search_pattern` or `search_patterns`? 

```python
>>> from colourlovers import clapi
>>> cl = clapi.ColourLovers()      
>>> cl.search_patterns(request='random')
[<colourlovers.data_containers.Pattern object at 0x0361D8B0>]
>>> cl.search_patterns(request='random')[0]
<colourlovers.data_containers.Pattern object at 0x0361DA60>
```

**Comment by flopp** [comment]
Is this already solved by #28?

**Comment by juangallostra** [comment]
@robintwhite @flopp definitely. I didn't publish a new release after the fix and hence the PyPi version of the wrapper includes the bug. Will fix it right now.

**Comment by juangallostra** [comment]
@robintwhite just published a new release with @flopp 's fix. Upgrading the package to version 0.2.0 should fix the issue.

**Comment by juangallostra** [comment]
Fixed as per #28, release 

@robintwhite feel free to reopen the issue of the bug is still present after upgrading to [v.0.2.0](https://github.com/juangallostra/Colourlovers-API-wrapper/releases/tag/0.2.0)

**Comment by robintwhite** [comment]
thanks - this works now in the new version on pypi
